symfony/plugins/orangehrmAdminPlugin/Api/PayGradeAllowedCurrencyAPI.php

```diff
@@ -0,0 +1,104 @@
+<?php
+/**
+ * OrangeHRM Enterprise is a closed sourced comprehensive Human Resource Management (HRM)
+ * System that captures all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM Inc is the owner of the patent, copyright, trade secrets, trademarks and any
+ * other intellectual property rights which subsist in the Licensed Materials. OrangeHRM Inc
+ * is the owner of the media / downloaded OrangeHRM Enterprise software files on which the
+ * Licensed Materials are received. Title to the Licensed Materials and media shall remain
+ * vested in OrangeHRM Inc. For the avoidance of doubt title and all intellectual property
+ * rights to any design, new software, new protocol, new interface, enhancement, update,
+ * derivative works, revised screen text or any other items that OrangeHRM Inc creates for
+ * Customer shall remain vested in OrangeHRM Inc. Any rights not expressly granted herein are
+ * reserved to OrangeHRM Inc.
+ *
+ * Please refer http://www.orangehrm.com/Files/OrangeHRM_Commercial_License.pdf for the license which includes terms and conditions on using this software.
+ *
+ */
+
+
+namespace OrangeHRM\Admin\Api;
+
+
+use OrangeHRM\Admin\Api\Model\CurrencyTypeModel;
+use OrangeHRM\Admin\Dto\PayGradeCurrencySearchFilterParams;
+use OrangeHRM\Admin\Service\PayGradeService;
+use OrangeHRM\Core\Api\CommonParams;
+use OrangeHRM\Core\Api\V2\CollectionEndpoint;
+use OrangeHRM\Core\Api\V2\EndpointCollectionResult;
+use OrangeHRM\Core\Api\V2\Endpoint;
+use OrangeHRM\Core\Api\V2\EndpointResult;
+use OrangeHRM\Core\Api\V2\ParameterBag;
+use OrangeHRM\Core\Api\V2\RequestParams;
+use OrangeHRM\Core\Api\V2\Validator\ParamRule;
+use OrangeHRM\Core\Api\V2\Validator\ParamRuleCollection;
+use OrangeHRM\Core\Api\V2\Validator\Rule;
+use OrangeHRM\Core\Api\V2\Validator\Rules;
+use OrangeHRM\Core\Traits\ServiceContainerTrait;
+use OrangeHRM\Framework\Services;
+
+class PayGradeAllowedCurrencyAPI extends Endpoint implements CollectionEndpoint
+{
+    use ServiceContainerTrait;
+
+    /**
+     * @return PayGradeService
+     * @throws \Exception
+     */
+    public function getPayGradeService(): PayGradeService
+    {
+        return $this->getContainer()->get(Services::PAY_GRADE_SERVICE);
+    }
+
+    public function getAll(): EndpointResult
```

Comment:
missing doc blocks in all public functions

Reply:
Done